MH4U Dex/Constants.h

```diff
@@ -0,0 +1,115 @@
+//
+//  Constants.h
+//  MH4U Dex
+//
+//  Created by Joseph Goldberg on 3/18/15.
+//  Copyright (c) 2015 Joseph Goldberg. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface Constants : NSObject
+
+FOUNDATION_EXPORT NSString *const JSON;
+FOUNDATION_EXPORT NSString *const TrueString;
+FOUNDATION_EXPORT NSString *const LowConstString;
+FOUNDATION_EXPORT NSString *const HighConstString;
+FOUNDATION_EXPORT NSString *const GConstString;
+
+#pragma mark - Item Constants
+
+FOUNDATION_EXPORT NSString *const ItemNameKey;
+FOUNDATION_EXPORT NSString *const ItemEncyclopedia;
+
+#pragma mark - Monster Constants
+
+FOUNDATION_EXPORT NSString *const MonsterIDKey;
+FOUNDATION_EXPORT NSString *const MonstersFileName;
+FOUNDATION_EXPORT NSString *const MonsterDamageZonesFileName;
+FOUNDATION_EXPORT NSString *const MonsterEncyclopedia;
+FOUNDATION_EXPORT NSString *const Boss;
+FOUNDATION_EXPORT NSString *const Minion;
+
+#pragma mark - Area Constants
+
+FOUNDATION_EXPORT NSString *const AreaCombinedNameKey;
+
+#pragma mark - Region Constants
+
+FOUNDATION_EXPORT NSString *const RegionsFileName;
+FOUNDATION_EXPORT NSString *const RegionIDJSONKey;
+FOUNDATION_EXPORT NSString *const RegionNameKey;
+FOUNDATION_EXPORT NSString *const RegionKeyNameKey;
+FOUNDATION_EXPORT NSString *const RegionDropsFileNameSuffix;
+FOUNDATION_EXPORT NSString *const RegionEncyclopedia;
+
+#pragma mark - Quest Constants
+
+FOUNDATION_EXPORT NSString *const QuestsFileName;
+FOUNDATION_EXPORT NSString *const QuestIDKey;
+FOUNDATION_EXPORT NSString *const QuestNameKey;
+FOUNDATION_EXPORT NSString *const QuestCaravanHallKey;
+FOUNDATION_EXPORT NSString *const QuestDangerKey;
+FOUNDATION_EXPORT NSString *const QuestFeeKey;
+FOUNDATION_EXPORT NSString *const QuestHRPKey;
+FOUNDATION_EXPORT NSString *const QuestMapKeyNameKey;
+FOUNDATION_EXPORT NSString *const QuestKeyIndicatorKey;
+FOUNDATION_EXPORT NSString *const QuestObjectiveKey;
+FOUNDATION_EXPORT NSString *const QuestRewardKey;
+FOUNDATION_EXPORT NSString *const QuestStarsKey;
+FOUNDATION_EXPORT NSString *const QuestSubQuestObjectiveKey;
+FOUNDATION_EXPORT NSString *const QuestFirstMonsterKey;
+FOUNDATION_EXPORT NSString *const QuestSecondMonsterKey;
+FOUNDATION_EXPORT NSString *const QuestThirdMonsterKey;
+FOUNDATION_EXPORT NSString *const QuestFourthMonsterKey;
+FOUNDATION_EXPORT NSString *const QuestTypeKey;
+FOUNDATION_EXPORT NSString *const QuestUrgentKey;
+FOUNDATION_EXPORT NSString *const QuestCaravanKey;
+FOUNDATION_EXPORT NSString *const QuestNoneString;
```

Comment:
Since I haven't finished setting up the quest-display, I'm not sure what will only be used in one file and what will be used in more.  I'll go back to move things that only get used in one file to their own file once quest-display is finished.